### DISCUSSION TITLE
Search path does not exist error

### DISCUSSION THREAD

**Discussion by At1a** [KSP-ModularManagement/KSPe#52] [Support] [answered]
screenshot of error and log file attached. help would be greatly appreciated

[KSP.log](https://github.com/net-lisias-ksp/KSPe/files/10836059/KSP.log)
![Screenshot (11)](https://user-images.githubusercontent.com/64880830/221474074-cdf348ba-91eb-4234-ab05-4de9a1449a05.png)

**Comment by Lisias** [comment]
Hi. @At1a ! Sorry the delay, I got here by plain luck due another issue. Next time, please mention me on the issue using @Lisias , so github will warn me about.

About your case, you have your GameData messed up:

```
[LOG 21:21:07.319] AssemblyLoader: Loading assembly at F:\SteamLibrary\steamapps\common\Kerbal Space Program\GameData\GameData\ModuleManager.4.1.4.dll
```

You see, there's two `GameData` in your rig, one inside another!

Move everything from `GameData/GameData` to `GameData`, then delete the empty `GameData/GameData` and everything will be ok!

Cheers!

**Comment by At1a** [comment]
thank you! i did that but still the same error. here is the [KSP.log](https://github.com/net-lisias-ksp/KSPe/files/10866929/KSP.log)

**Comment by Lisias** [comment, marked as answer]
Now it's a missing dependency, that it's triggering the infamous `Assembly Loader/Resolver` bug inside KSP.

```
[WRN 19:07:04.089] AssemblyLoader: Assembly 'HangerExtenderExtended' has not met dependency 'ToolbarController' V1.0.0
[WRN 19:07:04.089] AssemblyLoader: Assembly 'HangerExtenderExtended' is missing 1 dependencies
```

Install Toolbar Controller, or uninstall HangerExtenderExtended. This should fix your issue.

**Comment by Lisias** [comment]
Closing this ticket by lack of feedback from the user.
